client/js/lounge.js

```diff
@@ -814,4 +811,7 @@ $(function() {
 			});
 		}
 	});
+
+	// Only start opening socket.io connection after all events have been registered
+	socket.open();
```

Comment:
Any specific reason for this change, or is it just to be totally in agreement with the comment above?

Reply:
Align with the comment.

Reply:
To be fair, the statement in the comment is binary. You can't be partially in agreement with the comment. It is either true or it isn't.